hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieAppendHandle.java

```diff
@@ -194,8 +210,7 @@ private Option<FileSlice> populateWriteStatAndFetchFileSlice(HoodieRecord record
       prevCommit = instantTime;
       if (hoodieTable.getMetaClient().getTableConfig().isCDCEnabled()) {
         // the cdc reader needs the base file metadata to have deterministic update sequence.
-        TableFileSystemView.SliceView rtView = hoodieTable.getSliceView();
-        fileSlice = rtView.getLatestFileSlice(partitionPath, fileId);
+        fileSlice = getFileSlice();
```

Comment:
if CDC is not enabled and for table version 8, where are we populating the `fileSlice` ? 
I could not locate it.